server/planning/feed_parsers/superdesk_planning_xml.py

```diff
@@ -0,0 +1,194 @@
+from cgitb import text
+from datetime import date
+import logging
+from eve.utils import config
+from superdesk.io.feed_parsers import NewsMLTwoFeedParser
+import pytz
+from xml.etree.ElementTree import Element
+from flask import current_app as app
+from superdesk import get_resource_service
+from superdesk.metadata.item import (
+    CONTENT_TYPE,
+    ITEM_TYPE,
+    GUID_FIELD,
+    CONTENT_STATE,
+)
+from superdesk.utc import utcnow, utc_to_local
+from superdesk.errors import ParserError
+
+utc = pytz.UTC
+logger = logging.getLogger(__name__)
+
+
+class PlanningMLParser(NewsMLTwoFeedParser):
+    """Superdesk event specific parser.
+
+    Feed Parser which can parse the Superdesk Planing feed and convert to internal Planning format,
+    but the firstcreated and versioncreated times are localised.
+    """
+
+    NAME = "planningml"
+    label = "Planning ML"
+
+    # map subject qcode prefix to scheme
+    # if value is None, no "scheme" is used, and name comes from qcode
+    # if value is not None, <name> element is used instead of qcode
+    SUBJ_QCODE_PREFIXES = {"subj": None}
+    missing_voc = None
+
+    def can_parse(self, xml: Element):
+        self.root = xml
+        try:
+            if not xml.tag.endswith("planningItem"):
+                return False
+
+            item_meta = xml.find(self.qname("itemMeta"))
+            if item_meta is None:
+                return False
+
+            itemClass_node = item_meta.find(self.qname("itemClass"))
+            if itemClass_node is None:
+                return False
+
+            return itemClass_node.attrib.get("qcode", "") == "plinat:newscoverage"
+        except Exception:
+            return False
+
+    def parse(self, tree: Element, provider=None):
+        self.root = tree
+
+        try:
+            guid = tree.attrib["guid"]
+            item = {
```

Comment:
Should probably add a check for existing Planning item here, as supporting updates is troublesome and we need to come up with a plan for this.

Similar to EventsML: https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/feed_parsers/events_ml.py#L76